src/amp-story-player.js

```diff
@@ -171,6 +171,14 @@ export class AmpStoryPlayer {
       this.initializeHandshake_(story, iframeEl).then(
         messaging => {
           messaging.setDefaultHandler(() => Promise.resolve());
+
+          messaging.registerHandler('selectDocument', (event, data) => {
+            if (data.next) {
+              this.next_();
+            } else if (data.previous) {
+              this.previous_();
+            }
+          });
```

Comment:
Nit: Let's start extracting the handlers to different methods, as they might grow a lot (`onSelectDocument_()`)